presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -308,9 +308,14 @@ primaryExpression
     | '(' expression ')'                                                                  #parenthesizedExpression
     ;
 
+string
+    : STRING
+    | ESCAPE_INTRODUCER STRING
```

Comment:
I think this allows spaces between the 'U&' and the quote which isn't allowed by 5.3:
```
11) In a <Unicode character string literal>, there shall be no <separator> between the “U”
and the <ampersand> nor between the <ampersand> and the <quote>.